src/main/java/perkpack/models/User.java

```diff
@@ -0,0 +1,93 @@
+package perkpack.models;
+
+import org.hibernate.validator.constraints.Email;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Objects;
+
+@Entity
+public class User {
+
+    @NotNull
+    @Size(min =1, max = 32)
+    private String firstName;
+
+    @NotNull
+    @Size(min =1, max = 32)
+    private String lastName;
+
+    @NotNull
+    @Email
+    @Size(min =1, max = 32)
+    private String email;
+
+    @Id
+    @GeneratedValue
+    private long id;
+
+    public User()
```

Comment:
We don't need this for H2 DB.

Reply:
Just found out that the tests rely on his empty constructor.